From 662157070aad6a238ee0089d5f0bb55ab8e03908 Mon Sep 17 00:00:00 2001
From: llenotre <llenotre@student.42.fr>
Date: Thu, 8 Apr 2021 00:39:26 +0200
Subject: [PATCH] Change EXTRAVERSION to '-thor_kernel'

Signed-off-by: llenotre <llenotre@student.42.fr>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index cc77fd45ca64..6ee8318cefe6 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 12
 SUBLEVEL = 0
-EXTRAVERSION = -rc6
+EXTRAVERSION = -thor_kernel
 NAME = Frozen Wasteland
 
 # *DOCUMENTATION*
-- 
2.25.1

